src/components/DevDummyLogin/index.js

```diff
@@ -61,7 +61,7 @@ const S = StyleSheet.create({
 
 const DevDummyLogin = ({resetClickTime = () => {}}) => {
   const {ENABLE_DEV_ONLY_FEATURE} = configEnv;
-  const {setProfileId} = useProfileHook();
+  const {setAuth} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setAuth` function from `useUserAuthHook` is being used instead of `setProfileId` from `useProfileHook`. This change aligns with the previous comment about the shift towards using `useUserAuthHook`.
```diff
- const {setProfileId} = useProfileHook();
+ const {setAuth} = useUserAuthHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->